src/main/java/org/kohsuke/github/GHDeployment.java

```diff
@@ -89,6 +92,16 @@ public Object getPayloadObject() {
         return payload;
     }
 
+    /**
+     * The environment defined when the deployment was first created.
+     *
+     * @return the original deployment environment
+     */
+    @Preview(Previews.FLASH)
```

Comment:
We mark preview fields as `@Deprecated` to make sure end users are aware (via editor feedback) that they are not stable APIs. 

This comment applies to all usages. 

```suggestion
    @Deprecated
    @Preview(Previews.FLASH)
```